common/src/main/java/com/lightstep/tracer/shared/AbstractTracer.java

```diff
@@ -477,7 +454,7 @@ private boolean sendReportWorker(boolean explicitRequest) {
             }
 
             // TODO should probably prepend and do it in bulk
-            for (SpanRecord span : req.span_records) {
+            for (Span span : reqBuilder.getSpansList()) {
```

Comment:
(to be clear, fine to leave it this way for now... I'm just trying to point out things where the Java client is maybe showing its age a bit)